### PR TITLE
CartoDB -> CARTO

### PR DESCRIPTION
**Pull request by javitonino**
Changes
screenshot_20160804_135548
to
screenshot_20160804_135614

### CODE REVIEW

**Comment by javitonino** [comment]
Please CR carefully, this is a dangerous change! @gfiorav

File: app/views/admin/shared/_api_keys_subheader.erb [hunk 1]
```diff
@@ -7,7 +7,7 @@
         <ul class="Filters-group">
           <li class="Filters-typeItem">
             <a href="<%= CartoDB.url(self, 'api_key_credentials', {}, current_user) %>" class=" CDB-Text CDB-Size-medium Filters-typeLink <%= 'is-selected' if "#{params[:controller]}_#{params[:action]}" == "admin/client_applications_api_key" %>">
-              CartoDB
+              CARTO
```
Comment (gfiorav) [inline app/views/admin/shared/_api_keys_subheader.erb]: I'm actually quite concerned with this change. Reasons:

+ No test is verifying this change.
+ We're passing from having 7 letters to just 5
+ Previously, 3 were caps, now 5 (almost 75% increase)
+ No clear pattern is followed, suggesting unorganised code.

I'm not able to say this is _safe_. I'd call for someone else to leverage in, but i'm pretty confident this is a no-go.

**Comment by gfiorav** [comment]
Just kidding, 📦 it

File: app/views/admin/shared/_api_keys_subheader.erb (hunk 1 above)
Comment (juanignaciosl) [inline app/views/admin/shared/_api_keys_subheader.erb]: I'm certainly worried about the risk of people shouting "CARTO" in front of their laptops with more people around. The social impact can be high.

Comment (javitonino) [inline app/views/admin/shared/_api_keys_subheader.erb]: What about `<span style="text-transform: uppercase;">carto</span>`. We even reduce the number of caps like way and there is no way any sane human will think of yelling with that [citation needed].

Comment (juanignaciosl) [inline app/views/admin/shared/_api_keys_subheader.erb]: That way you'll make frontenders yell, because of the `style` attribute.

**Pull request merged**